Mathlib/Algebra/Ring/Equiv.lean

```diff
@@ -77,18 +77,13 @@ add_decl_doc RingEquiv.toMulEquiv
 /-- `RingEquivClass F R S` states that `F` is a type of ring structure preserving equivalences.
 You should extend this class when you extend `RingEquiv`. -/
 class RingEquivClass (F R S : Type*) [Mul R] [Add R] [Mul S] [Add S] [EquivLike F R S] : Prop
```

Comment:
I don't believe we've come to a conclusion about what will happen with `RingEquivClass`. The precise form we end up using is not the most important here to me: I want to know that there is some sort of plan. Because stopping here means we become very inconsistent: -`EquivClass`es for everything but `Add`- and `Mul`-.